config/initializers/array_paginate.rb

```diff
@@ -0,0 +1 @@
+require 'will_paginate/array'
```

Comment:
Nice one 👌Had the same issue few days ago in Femtasy, Im using Kaminari there so the resolution was a little bit different. Anyway, gj.